src/transformers/quantizers/quantizer_compressed_tensors.py

```diff
@@ -113,3 +160,7 @@ def is_qat_trainable(self) -> bool:
     def is_serializable(self, safe_serialization=None) -> bool:
         """Models quantized using compressed tensors can be saved to disk"""
         return True
+
+    def dequantize(self, model: "PreTrainedModel"):
```

Comment:
do we have to call this dequantize to match huggingface API? If not `decompress` would be more accurate since it might involve something beyond quantization?